### PR TITLE
Prepare source for F722 targets

### PR DESCRIPTION
**Pull request by sambas**
Prepare source for F722 targets
Fix and combine F7 pwm output
Use flash split for F7 targets, can be updated for F4 also

### CODE REVIEW

File: src/main/target/link/stm32_flash_split.ld [hunk 1]
```diff
@@ -0,0 +1,162 @@
+/*
```
Comment (digitalentity) [inline src/main/target/link/stm32_flash_split.ld:1]: Why do we need a separate linker script?

Comment (sambas) [reply src/main/target/link/stm32_flash_split.ld:1]: this is bf style to use smaller lower sectors for config

Comment (digitalentity) [reply src/main/target/link/stm32_flash_split.ld:1]: I see. The first page is only occupied by ISR vector table, larger code occupies bigger flash pages. Smart solution.

**Comment by digitalentity** [comment]
Will have a better look tomorrow - too sleepy today :smile:

**Comment by sambas** [comment]
configurator needs this https://github.com/betaflight/betaflight-configurator/commit/e4794221b95c8a682b790e8f23148984479f87aa

**Comment by sambas** [comment]
Should be safe to merge, tested changes on 745 and 722 target. Split flash for F4 can be done later.

**Review by digitalentity — APPROVED** [review]

**Pull request merged**